src/core/Cargo.toml

```diff
@@ -13,6 +13,7 @@ workspace = "../.."
 [features]
 #TODO: switch to pure `serde` feature once `draw_state` is no longer used
 serialize = ["serde", "serde_derive", "draw_state/serialize"]
+copy = []
```

Comment:
Have you planned further portability specific changes? If so, we might want to rename this feature as it's probably only needed for FFI related tasks.